lightning/src/onion_message/packet.rs

```diff
@@ -147,7 +148,7 @@ impl<T: OnionMessageContents> Writeable for ParsedOnionMessageContents<T> {
 }
 
 /// The contents of an onion message.
-pub trait OnionMessageContents: Writeable {
+pub trait OnionMessageContents: Writeable + core::fmt::Debug {
```

Comment:
Looks like this is causing the fuzz tests to fail. You'll need to update that as well. You can run the tests with `cd fuzz && RUSTFLAGS="--cfg=fuzzing"`.